website/blog/2022-05-10-acf-in-a-project.md

```diff
@@ -0,0 +1,140 @@
+---
+title: ACF in a project
+description: Example of using ACF in your project
+slug: acf-in-a-project
+authors: obradovic
+date: 2022-05-10
+tags: [eightshift, boilerplate, acf, advanced custom fields, theme options]
+hide_table_of_contents: false
+---
+
+If you've worked with WordPress for long enough, you've heard of the **_Advanced Custom Fields_** plugin (**_ACF_** for short). While the use of blocks has simplified content editing, making meta fields less of a necessity, there are still cases in which meta fields are very useful.
+<!--truncate-->
+
+## Ways of registering ACF fields
+
+There are multiple approaches to ACF field registration. The easiest way to register fields is using the WP Admin interface. However, if you're using Git, this may not be the best way to do it since the fields are being saved in the database, which means you would have to export and import fields across environments each time you make a change.
```

Comment:
> However, if you're using Git, this may not be the best way to do it

This part makes no sense. 

Using Git has no effect on the database, or why we are using our approach.

I get what you're trying to say, but Git has nothing to do with this.

The main drawback (besides the import/export of the fields) is that when you have a large number of fields the admin can become sluggish, and fetching fields registration via database can impact performance.

Then, in the next paragraph, I'd suggest the other ways of field registration: either via PHP or using JSON, and then say that we use PHP because PHP can use opcache to speed things up even more. And then you can mention that registering fields in the PHP is better because you can commit those changes in Git for easier portability.

Reply:
I believe @iobrado's point here was that ACF fields that are registered using the database can't be easily versioned using version control software. 

Your point about optimization still stands, and his point might be a bit unclear, but the idea that was communicated, from what I've gathered from the post, is that you want versioned fields for maintainability and collaboration, and field definitions as code so your theme isn't configuration-dependent.

This makes perfect sense to me, but could be a bit clearer in the post 🤷

Reply:
I kinda see what he wanted to say, but the wording is odd/wrong so he should just reword this part and the second paragraph according to the suggestions 🙂